src/modules/Packages.rb

```diff
@@ -2558,17 +2489,23 @@ def patterns_to_install
       patterns.uniq
     end
 
-    def report_missing_pattern(pattern_name)
-      if (default_patterns | resolvable_mandatory_patterns).include?(pattern_name)
-        log.error "Mandatory pattern #{pattern_name} does not exist"
-        # Error message, %{pattern_name} is replaced with the missing pattern name in runtime
-        Report.Error(_(
-          "Failed to select default product pattern %{pattern_name}.\n" \
-          "Pattern has not been found."
-        ) % { pattern_name: pattern_name })
-      else
-        log.info "Optional pattern #{pattern_name} does not exist, skipping..."
-      end
+    # @return [void]
+    def report_missing_patterns(pattern_names)
+      return if pattern_names.empty?
+
+      # "p1, p2, p3, p4, p5,\n" \
+      # "p6, p7, p8"
+      names_s = pattern_names.each_slice(5).map { |n5| n5.join ", " }.join ",\n"
+      # %s is a list of pattern names
+      Report.Error(_(
+        "Failed to select default product patterns:\n" \
+        "%s\n" \
+        "Patterns have not been found.\n" \
+        "\n" \
+        "This can be probably be fixed by adding\n" \
```

Comment:
Sorry that I caught this only now but there is a duplication of "be" here :(

I hope my fix in https://github.com/yast/yast-packager/pull/405 is acceptable